packages/containers/src/ConfirmDialog/ConfirmDialog.container.js

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes } from 'react';
+import { Map } from 'immutable';
+import { ConfirmDialog as Component } from 'react-talend-components';
+
+import { getActionsProps } from '../actionAPI';
+import { statePropTypes, initState } from '../state';
+
+export const DEFAULT_STATE = new Map({
+	show: false,
+	validateAction: {
+		label: 'Ok',
+		bsStyle: 'primary',
+		onClick: () => {},
+	},
+	cancelAction: {
+		label: 'No',
+		onClick: () => {},
+	},
+});
```

Comment:
validateAction and cancelAction are not useful as state and do not reprensent the future action.
it must be a string, so they should not exists as state.